BREEZE.rst

```diff
@@ -717,33 +709,42 @@ On the host:
   ./scripts/ci/ci_pylint.sh ./airflow/example_dags/test_utils.py
 
 Running Test Suites via Scripts
---------------------------------------------
+-------------------------------
+
+To run tests use ``./scripts/ci/local_ci_run_airflow_testing.sh`` scripts.
+
+Optionally use one of the variables (defaults are bold):
+
+* PYTHON_VERSION is one of **3.5**/3.6/3.7
+* BACKEND is one of **sqlite**/postgres//mysql
+
+For Kubernetes tests you should set ``RUN_KUBERNETES_TESTS`` variable to true.
+This will start Kubernetes cluster inside the container using kind (Kubernetes-IN-Docker).
```

Comment:
```
-This will start Kubernetes cluster inside the container using kind (Kubernetes-IN-Docker).
+This will start Kubernetes cluster inside the container using [kind (Kubernetes-IN-Docker)](https://kind.sigs.k8s.io/).
```

Reply:
Indeed.